api/api.go

```diff
@@ -106,7 +108,10 @@ func (h *httpHandler) initRouter() {
 			{"/services", []string{"partial-id", epPIDQr}, procGetServices},
 			{"/services", nil, procGetServices},
 			{"/services/" + epID, nil, procGetService},
-			{"/services/" + epID + "/backend", nil, procGetContainers},
+			{"/services/" + epID + "/backend", nil, procGetSandbox},
```

Comment:
Now that we are introducing sandbox as the top-level resource, Do we need to expose this `backend` resource any more ?